modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -232,6 +232,15 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="expandInternalEntities" type="mule:substitutableBoolean" default="true">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            Whether to expand internal entity references in the XML.
+                            Default value is true. Keeping this value will make your application
```

Comment:
Again, I think we should by default NOT be vulnerable and inform here of the problem with changing the value to true. Since this is a change in behaviour from 3.8 you also need to add an entry to the migration guide (MIGRATION.txt).